src/datastation/scripts/fix_provenance_files.py

```diff
@@ -22,10 +22,11 @@
                      'xmlns:abr="http://www.den.nl/standaard/166/Archeologisch-Basisregister/" ' \
                      'xmlns:ddm="http://easy.dans.knaw.nl/schemas/md/ddm/"'
 
+provenance_schema_doc = etree.parse('http://easy.dans.knaw.nl/schemas/bag/metadata/prov/provenance.xsd')
 
-def validate(xml_path: str, xsd_path: str) -> bool:
-    xmlschema_doc = etree.parse(xsd_path)
-    xmlschema = etree.XMLSchema(xmlschema_doc)
+
+def validate(xml_path: str) -> bool:
+    xmlschema = etree.XMLSchema(provenance_schema_doc)
```

Comment:
Don't you want to reuse `xmlschema` across calls?

Reply:
done